src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -153,6 +153,15 @@ def is_sku_available(cmd, sku_info, zone):
     return is_available
 
 
+def is_empty_or_image_os_disk(DiskCreateOption, disk_create_option, os_type):
```

Comment:
Personally, I think it will be easier to understand if the method is named as business related
```suggestion
def need_to_enable_trust_launch(DiskCreateOption, disk_create_option, os_type):
```